### PR TITLE
media-sound/gmusicbrowser: bump to 1.1.16 version

### PR DESCRIPTION
**Pull request by bor**

### CODE REVIEW

File: media-sound/gmusicbrowser/gmusicbrowser-9999.ebuild [hunk 1]
```diff
@@ -5,26 +5,21 @@ EAPI=7
...
-	SRC_URI=""
-else
-	GIT_COMMIT="4538a5af5fb6c11f07bd7a9c50d6fd73b18c840d"
-	SRC_URI="https://github.com/squentin/${PN}/archive/${GIT_COMMIT}.tar.gz -> ${P}.tar.gz"
-	KEYWORDS="~amd64 ~x86"
-	S="${WORKDIR}/${PN}-${GIT_COMMIT}"
-fi
-
 DESCRIPTION="An open-source jukebox for large collections of mp3/ogg/flac files"
 HOMEPAGE="https://gmusicbrowser.org/"
```
Comment (vaukai) [inline media-sound/gmusicbrowser/gmusicbrowser-9999.ebuild:10]: remove empty line here

File: media-sound/gmusicbrowser/gmusicbrowser-9999.ebuild [hunk 2]
```diff
@@ -5,26 +5,21 @@ EAPI=7
...
 
+EGIT_REPO_URI="https://github.com/squentin/${PN}.git"
+inherit git-r3
```
Comment (vaukai) [inline media-sound/gmusicbrowser/gmusicbrowser-9999.ebuild:12]: Move this inherit up to the first inherit

File: media-sound/gmusicbrowser/gmusicbrowser-9999.ebuild [hunk 3]
```diff
@@ -5,26 +5,21 @@ EAPI=7
...
+
+SRC_URI=""
+KEYWORDS=""
+
```
Comment (vaukai) [inline media-sound/gmusicbrowser/gmusicbrowser-9999.ebuild:16]: remove empty SRC_URI and KEYWORDS

File: media-sound/gmusicbrowser/gmusicbrowser-1.1.16.ebuild [hunk 4]
```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils gnome2-utils xdg
+
+DESCRIPTION="An open-source jukebox for large collections of mp3/ogg/flac files"
+HOMEPAGE="https://gmusicbrowser.org/"
+
```
Comment (vaukai) [inline media-sound/gmusicbrowser/gmusicbrowser-1.1.16.ebuild:10]: remove empty line here

File: media-sound/gmusicbrowser/gmusicbrowser-1.1.16.ebuild [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+SRC_URI="https://${PN}.org/download/${P}.tar.gz"
+KEYWORDS="~amd64 ~x86"
```
Comment (vaukai) [inline media-sound/gmusicbrowser/gmusicbrowser-1.1.16.ebuild:12]: put empty line under SRC_URI and move KEYWORDS down between SLOT and IUSE

File: media-sound/gmusicbrowser/gmusicbrowser-1.1.16.ebuild [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+			media-sound/alsa-utils
+			media-sound/flac123
+			|| ( media-sound/mpg123 media-sound/mpg321 )
+			media-sound/vorbis-tools
+		)
+	)
+	extras? ( dev-perl/gnome2-wnck )
+	libnotify? ( dev-perl/Gtk2-Notify )"
+DEPEND="sys-devel/gettext
+	doc? ( dev-perl/Text-Markdown )"
```
Comment (vaukai) [inline media-sound/gmusicbrowser/gmusicbrowser-1.1.16.ebuild:35]: Look for "gettext" in the source code.  If gettext is used for compilation it should be moved to BDEPEND.

Comment (bor) [reply media-sound/gmusicbrowser/gmusicbrowser-1.1.16.ebuild:35]: Thank you for the review, @vaukai !
I've applied all your suggestion above.

About this case - indeed, there is no direct usage of `gettext` in the code.
Only `Makefile` (and sub calls of it) uses `msgfmt`, `msgmerge`, etc.
So I'll move the dep.

BTW I've checked the remaining dependency and found that it also used only in the build process. So I'll move both to `BDEPEND`.

File: media-sound/gmusicbrowser/gmusicbrowser-1.1.16.ebuild [hunk 7]
```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils gnome2-utils xdg
```
Comment (vaukai) [inline media-sound/gmusicbrowser/gmusicbrowser-1.1.16.ebuild:6]: Does it really need eutils? If not, remove it

Comment (bor) [reply media-sound/gmusicbrowser/gmusicbrowser-1.1.16.ebuild:6]: yep, `einstalldocs` is used

Comment (bor) [reply media-sound/gmusicbrowser/gmusicbrowser-1.1.16.ebuild:6]: I see the eclass marked as deprecated now. So if you suggest alternative I can drop it.

File: media-sound/gmusicbrowser/gmusicbrowser-1.1.16.ebuild (hunk 6 above)
Comment (bor) [reply media-sound/gmusicbrowser/gmusicbrowser-1.1.16.ebuild]: Done

File: media-sound/gmusicbrowser/gmusicbrowser-1.1.16.ebuild (hunk 7 above)
Comment (ionenwks) [reply media-sound/gmusicbrowser/gmusicbrowser-1.1.16.ebuild:6]: einstalldocs is available in EAPI 6-7 without needing to inherit eutils, the eclass omits defining it if not EAPI 5 or lower

Comment (bor) [reply media-sound/gmusicbrowser/gmusicbrowser-1.1.16.ebuild]: Thank you for the hint.
I've removed it.

**Comment by fordfrog** [comment]
looks there is a qa issue that has to be resolved yet:

```
 * QA Notice: This package installs one or more .desktop files that do not
 * pass validation.
 * 
 * 	/usr/share/applications/gmusicbrowser.desktop: warning: key "OnlyShowIn" in group "Desktop Action LockAlbum" is deprecated
 * 	/usr/share/applications/gmusicbrowser.desktop: warning: key "OnlyShowIn" in group "Desktop Action LockArtist" is deprecated
 * 	/usr/share/applications/gmusicbrowser.desktop: warning: key "OnlyShowIn" in group "Desktop Action Next" is deprecated
 * 	/usr/share/applications/gmusicbrowser.desktop: warning: key "OnlyShowIn" in group "Desktop Action PlayPause" is deprecated
 * 	/usr/share/applications/gmusicbrowser.desktop: warning: key "OnlyShowIn" in group "Desktop Action Previous" is deprecated
 * ```

**Comment by fordfrog** [comment]
@bor could you please have a look at the qa issue above?

**Comment by bor** [comment]
> @bor could you please have a look at the qa issue above?

@fordfrog please check
I've fixed that, rebased on last master as well.

**Comment by fordfrog** [comment]
@bor cool, thanks!

**Pull request closed without merging**